index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <div id="navbar">
+  <a class="active" href="#home">Home</a>
+  <a href="#news">News</a>
+</div>
+  <section id="welcome-section">
+<h1>welcome</h1>
+</section>
+<section id="projects">
+  <p1 class="project-tile">this is mohammed and roland</p1>
+  <a>this is our project</a>
+</section>
+<a id="profile-link" target="_blank"
+</a>
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
```

Comment:
These lines are misplaced. please try to move them exactly before your nav bar. Everything that will be displayed on your page should be inside the body element. Therefore, all your code starting with the navbar should be after this and not before